### ISSUE TITLE
Unclear how to login with app

### ISSUE THREAD

**Issue by turnercore** [balzack/databag#79]
I'm pretty tech savvy and it was not clear at all how to login to my server from the iOS app. 

For any looking for the answer to this you login with your username@server-domain.com, with no http/https in front. 

I don't know why the way to login would be username@server.com and the placeholder text says username / server, think this needs to be fixed for people to be able to use the app.

**Comment by balzack** [comment]
Yes, I agree it's not clear. The login username/server.com should work, but username@server.com should also work.

Initially the system login was username@server.com, but people pointed out this was confusing being the same format as email.

Support was added for '/' while '@' is deprecated. Please let me know if the '/' doesn't work. I plan on improving the messaging for the login in the next release.

Thanks for your interest in the project!

**Comment by jinlaohu** [comment]
I can't login using android databag app. It always say Error please try again.  Whether i use username@server.com or username/server.com, it dosen't work.

**Comment by cesheji3** [comment]
The same to me.Please fix the bugs.

**Comment by ecker00** [comment]
I am trying to sign in to an instance hosted on a VPN network, not able to get `username/10.0.0.100:7000` to work in the app, guess the port number throws it off? I do plan to put it behind a reverse proxy eventually, but for initial testing felt natural that this should work.

While @ barrows the structure of email, I think that's not a bad thing, it's already familiar structure on Mastodon / ActivityPub.

**Comment by balzack** [comment]
The mobile app doesn't currently support IP login, you need a dns name; it should however work with the browser.

The reason IP isn't currently supported is that I disabled non-secure http from the mobile app. I think that might not be a requirement, so I plan to revisit that in the next release.

The '@' should still work in terms of logging in, but isn't displayed as the hint anymore. I agree there is also a similar conflict with '/'. Do you recommend sticking with '@', or can you think of another character?

Thanks for using this project and the feedback. I really appreciate it!

**Comment by CoreyCCarter** [comment]
I have tried both ways of logging in on the iPhone app and it still gives me the 'login error'. I am able to login on a web browser but not through the app.

**Comment by balzack** [comment]
Could you try creating a test account on https://databag.coredb.org, and seeing if the iOS app will connect there?

**Comment by CoreyCCarter** [comment]
I did a test earlier and it worked fine via the iOS app it just seems to be when I try it with my server it does not work.

**Comment by balzack** [comment]
hmm, are you using a dns name & cert?

In case you are trying to use it with just an IP address (no cert), that isn't yet supported in the mobile app.

**Comment by CoreyCCarter** [comment]
I deleted the app and reinstalled it and it works perfectly now! Thank You for being so responsive and so quick!

**Comment by balzack** [comment]
No worries....thanks for trying out this project!
